### DISCUSSION TITLE
build/make/tools/extract_kernel.py is failing to extract kernel information

### DISCUSSION THREAD

**Discussion by dogpoopy** [ROM-builders/temporary#22666] [General]
Hi Sir,

I'm facing an issue with the build/make/tools/extract_kernel.py script. The script is failing to extract kernel information.

https://cirrus-ci.com/task/5828130942746624?logs=build#L8340

I would be grateful if you could take a look at the problem and let me know if the failure was on my end.

Thank you for your time.

**Comment by Apon77** [comment]
It is from your side. Clean build is also showing the same error. 
![image](https://github.com/ROM-builders/temporary/assets/61594531/c34ff94f-8340-4939-8566-78574a1ef5bf)
